docs/using/administering/debugging.rst

```diff
@@ -33,6 +37,50 @@ It is possible to see the available unit names, and then view the logs with ``jo
    [root@node1]# journalctl -u flocker-container-agent
    [root@node1]# journalctl -u flocker-control
 
+Bug Reporting
+-------------
+
+When reporting issues with Flocker please include:
+
+* operating system and version,
+* Linux kernel version,
+* Docker version and configuration,
+* Flocker log files and
+* recent syslog content.
```

Comment:
`syslog` is inaccurate; on Ubuntu we write our log files out. Maybe "Recent Flocker log content"?